.gitignore

```diff
@@ -141,3 +141,6 @@ dmypy.json
 _build/
 
 systems.json
+
+# OSX generated files
+.DS_store
```

Comment:
It's worth putting this in your [global git `core.excludesFile`](https://stackoverflow.com/a/22885996/51685) instead.

Reply:
I don't see any argument against having it there so i'll leave it.

Reply:
I would argue this isn't part of the PR :)

The exclude file is a good idea to avoid you having to do this sort of thing for all projects you touch on a Mac.